packages/pentaho/src/Canvas/SidePanel/SidePanel.tsx

```diff
@@ -90,16 +108,29 @@ export const HvCanvasSidePanel = forwardRef<
     tabs?.[0]?.id ?? "none",
   );
 
+  useEffect(() => {
+    handleSidePanelWidth?.(initialWidth);
+    onResize?.(initialWidth);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [initialWidth]);
```

Comment:
`onResize` should only be called when the user actually resizes

```suggestion
  useEffect(() => {
    handleSidePanelWidth?.(initialWidth);
  }, [initialWidth, handleSidePanelWidth]);
```